solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -612,6 +656,18 @@ contract LiquidityProtection is ILiquidityProtection, TokenHandler, Utils, Owned
             );
         }
 
+        // notify event subscribers
+        if (address(eventsSubscriber) != address(0)) {
```

Comment:
called after removing liquidity (but before stats are updated) - intentional?

Reply:
Hmmm... it'd be cleaner to move it before the liquidity is removed, although it won't affect the current implementation